README.md

```diff
@@ -1,77 +1,111 @@
+  * Build metadata placed in ivy description section
+    - Also in jar manifests via the nebula.info plugin
+  * Build metadata mapped from `buildEnv` extension object
+    - Pre-populated with various default values from the environment
+    - Can map arbitrary values (expando object)
 
-* `publish` lifecycle is standard publish task aggregator
-    * Automatically depends on artifactoryPublish or bintrayUpload if applicable
-    * Supports the ivy-publish / maven-publish plugins
+[Artifactory Plugin]: https://www.jfrog.com/confluence/display/RTF/Gradle+Artifactory+Plugin
 
-* `install` will publish artifact to local filesystem if applicable
-    * Projects using `maven-publish` will hook Gradle's built-in task
-    * Ivy projects will look for a 'local' ivy repo to publish to
+## Optional Versioning Plugins
+There are two versioning convention plugins provided with the ReadyTalk CI
+plugin, but they need to be applied separately. For both of these plugins, it
+is reccommended the version be stored in `gradle.properties`, but other
+mechanisms should work - the version string is modified after the build
+script's configuration phase has been evaluated.
 
-##Conventions
+### Snapshot version plugin
+This plugin provides a Maven-style versioning conventions where '-SNAPSHOT' is
+appended to pre-release development build versions. With this plugin, you only
+need to store the version of the next intened release in the codebase.
 
- * Adds integTest task for integration tests
- * Auto-wire jacoco reporting into check phase
- * `ci` task auto-depends on `publish` if building master in a CI build
-     * Supports Travis CI and Jenkins
- * [Artifactory Plugin][] automatically includes publications by default
- * Build metadata placed in ivy description section
-     * Also in jar manifests via the nebula.info plugin
- * Build metadata mapped from `buildEnv` extension object
-     * Pre-populated with various default values from the environment
-     * Can map arbitrary values (expando object)
+To apply the Snapshot Version Plugin, make sure the CI plugin is already on the
+classpath (using either the "classic" way or the new `plugins{}` closure and
+add:
 
-[Artifactory Plugin]: https://www.jfrog.com/confluence/display/RTF/Gradle+Artifactory+Plugin
+```groovy
+apply plugin: 'com.readytalk.ci.version.snapshot'
+```
 
-###Usage
+### Buildnumber version plugin
+This plugin is intended to be used for projects using a continuous delivery
+pipeline with multiple test and deployment stages. The build number from the CI
+server is appended to the version string to each instance of the pipeline to
+use its own artifacts.
 
-####artifactoryPublish example:
+To apply the BuildNumber Version Plugin, make sure the CI plugin is already on
+the classpath (using either the "classic" way or the new `plugins{}` closure
+and add:
 
+```groovy
+apply plugin: 'com.readytalk.ci.version.buildnumber'
 ```
+
+### Usage
+
+#### artifactoryPublish example:
```

Comment:
In fact, I'd opt for not referencing buildEnv in the README at all, or only in an advanced section. There's a possibility it might get heavily refactored in the future if we pull in the [extended info plugin](https://github.com/boxheed/gradle-extended-info-plugin).